src/shared-components/chip/index.tsx

```diff
@@ -46,6 +46,16 @@ const getStyles = ({ isLowContrast, status, theme }: GetStylesProps) => {
     }
   }
 
+  if (status === 'info') {
```

Comment:
[nit] For consistency with status = PRIMARY with low contrast, can we separate this into two separate conditionals? As a wider engineering org, we tend to avoid nested conditionals wherever possible.